src/test/java/seedu/address/logic/commands/EditStudentCommandTest.java

```diff
@@ -18,34 +18,35 @@
 
 import seedu.address.commons.core.Messages;
 import seedu.address.commons.core.index.Index;
-import seedu.address.logic.commands.EditCommand.EditPersonDescriptor;
```

Comment:
missing replacement for this, need to include `import seedu.address.logic.commands.EditCommand.EditStudentDescriptor`